drivers/include/isl29020.h

```diff
@@ -39,13 +39,13 @@ extern "C" {
 /**
  * @brief Default address
  *
- * The address depends on the status of A0 Pin.
- * Default address corresponds to A0 connected to GND.
- * For more information on SerialBus Address, refer Section I2C
- * Interface on Page 3 of datasheet.
+ * The address depends on the status of A0 Pin. Default
```

Comment:
Please only wrap lines to adjust to 80 chars.

Reply:
Done.